API.md

```diff
@@ -375,6 +376,39 @@ Txo Spent from One Account to Another in the Same Wallet
 * [get_all_txos_by_account](./README.md#get-all-txos-for-a-given-account)
 * [get_txo](./README.md#get-txo-details)
 
+
+### The Gift Code Object
+
+#### Attributes
+
+| *Name* | *Type* | *Description*
+| :--- | :--- | :---
+| gift_code | string | The b58 encoded contents of a gift code. Includes the entropy, the value, and a memo.
+
+#### More attributes
+
+| *Name* | *Type* | *Description*
+| :--- | :--- | :---
+| object | string, value is "gift_code" | String representing the object's type. Objects of the same type share the same value
+
+#### Example Object
+
+```
+{
+  "object": "account",
+  "gift_code": "1916a9b3...",
+  "entropy": "1916a9b3...",
+  "value": "2000000000000",
```

Comment:
Having the value feels questionable to me since there's no guarantee it actually matches the TxOut.